BParkingNano/python/recoBKll_cff.py

```diff
@@ -0,0 +1,74 @@
+import FWCore.ParameterSet.Config as cms
+from PhysicsTools.NanoAOD.common_cff import *
+
+
+process.reconstructB = cms.EDFilter(
+        "BKllProducer",
+         muons = cms.InputTag("muonsBPark:SelectedMuons"),
+         electrons = cms.InputTag("electronsBPark:SelectedElectrons"),
+         tracks = cms.InputTag("tracksBPark:SelectedTracks"),
+         beamSpot= cms.InputTag("offlineBeamSpot"),
+         RunParameters = cms.PSet(
+           SkipNoRecoBEvt = cms.bool(True),
+           FinalLeptonId = cms.int32(11),
+           label = cms.string("BKmumu"),
+           ptLep1Cut = cms.double(1.5),
+           bdtEl1Cut = cms.double(3.0),
+           dzLepLepCut = cms.double(1.0),
+           mllMax= cms.double(5.0),
+           mllMin= cms.double(0),
+           MBMin = cms.double(4.5),
+           MBMax = cms.double(6),
+           drTrkLepCut = cms.double(0.01),
+           BPtCut = cms.double(3),
+           BVtxProbCut = cms.double(0.001),
+           BCosACut = cms.double(0)
+         )
+)
+
+process.nanoSequence *= process.reconstructB
+
+
+
+process.bToKMuMuTable = cms.EDProducer(
```

Comment:
remove process.

Reply:
you are right. Some things need fixing in the cfg. Many apologies for that. I was testing the code by parts and forgot to remove the "process". Tomorrow I will rebase and update my PR